### ISSUE TITLE
Curl command fails if -u is specified

### ISSUE THREAD

**Issue by gpasq** [Huachao/vscode-restclient#26]
- VSCode Version: 1.3.1
- OS Version: Windows 10
- REST Client Version: 0.6.2

Steps to Reproduce:

1. Create a curl command that passes the -u flag with required parameters
2. run the curl, it will fail with "Unauthorized"

It appears that running the curl command does not process the -u parameters correctly.  You can reproduce this if you sign up for a twilio account and call:

curl "https://lookups.twilio.com/v1/PhoneNumbers/+19999999999?CountryCode=US&Type=carrier" -u 'account-SID:account-token'

Response is

```
HTTP/1.1 401 UNAUTHORIZED 317ms
access-control-allow-credentials: true
access-control-allow-headers: Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since
access-control-allow-methods: GET, POST, DELETE, OPTIONS
access-control-allow-origin: *
access-control-expose-headers: ETag
content-type: application/json
date: Sun, 31 Jul 2016 02:54:20 GMT
twilio-request-duration: 0.002
twilio-request-id: RQb1fa69f4011d45f1bd552952a270a886
www-authenticate: Basic realm="Twilio API"
x-powered-by: AT-5000
x-shenanigans: none
content-length: 205
connection: Close

{
    "code": 20003,
    "detail": "Your AccountSid or AuthToken was incorrect.",
    "message": "Authentication Error - No credentials provided",
    "more_info": "https://www.twilio.com/docs/errors/20003",
    "status": 401
}
```

**Comment by Huachao** [comment]
@gpasq, I have fixed in latest version 0.6.3, please help to verify

**Comment by gpasq** [comment]
That has fixed it @Huachao, thank you!
